contrib/examples/actions/workflows/customfilters/mistral-regex_match.yaml

```diff
@@ -0,0 +1,20 @@
+version: '2.0'
+
+examples.mistral-customfilters-regex_match:
+    description: A workflow for testing regex_match custom filter in mistral
+    type: direct
+    vars:
+        regex_pattern: "([0-9]{3} \\w+ (?:Ave|St|Dr))"
+    input:
+        - input_str
+    output:
+        result_jinja: <% $.result_jinja %>
+        result_yaql: <% $.result_yaql %>
+    tasks:
+
+        task1:
+            action: std.noop
+            publish:
+                result_jinja: '{{ _.input_str | regex_match(_.regex_pattern)}}'
```

Comment:
Alright, I have something working in https://github.com/StackStorm/st2mistral/pull/30/commits/cb874997441405d6d2b3a6d810e4ee03e509fc0c. I've tested both forms and it seems to work well.
Let me know what you think about the approach and I'll add the decorator to the rest of the filters in that PR and then add a new dedicated test in this PR for testing both forms on a few of the filters.